api/mm/API_VERSION

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
Version should be mentioned in changelog and documentation. Also, is there a mechanism for the regression tests to make reference to the API version choice?

Reply:
Done in 2f89d8bb